pandas/tests/util/test_util.py

```diff
@@ -530,3 +531,10 @@ def test_safe_import(monkeypatch):
     monkeypatch.setitem(sys.modules, mod_name, mod)
     assert not td.safe_import(mod_name, min_version="2.0")
     assert td.safe_import(mod_name, min_version="1.0")
+
+
+def test_lxml_import():
```

Comment:
can you rename to ``test_show_versions()`` and the issue number is not needed here. maybe just a comment that this test validations that show_versions() is accessing version info correctly.

Reply:
FWIW, I don't think this test is necessary. It wouldn't have caught the original bug, unless the environment had lxml but not openpyxl.

We also should be running `show_versions` as part of every build already right? Just for the output in the CI logs?

Reply:
ok, @TomAugspurger is right here, let's just remove this test entirely.

Reply:
@jreback removed the test case